components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -99,20 +99,33 @@ export class ReportingSearchbarComponent implements OnInit {
 
   onValChange(e) {
     const type = this.selectedType;
-    const value = this.filterValues.filter(v => v.title === e.target.value)[0];
+    const text: string = e.target.value;
+    const suggestionValue = this.filterValues.filter(v => v.title === text)[0];
 
-    if (value) {
-      this.filterAdded.emit({ detail: { type, value }});
+    if (this.containsWildcardChar(text)) {
+      const wildcardValue = {text, title: text};
 
-      this.selectedType = null;
-      this.keyInputVisible = true;
-      this.valInputVisible = false;
-
-      this.keyInput.nativeElement.value = '';
-      this.valInput.nativeElement.value = '';
+      this.addNewFilter(type, wildcardValue);
+    } else if (suggestionValue) {
+      this.addNewFilter(type, suggestionValue);
```

Comment:
The search bar term is only excepted if it matches a suggested value or has wildcard characters.